pkg/dwarf/godwarf/tree.go

```diff
@@ -24,11 +24,11 @@ func (ce compositeEntry) Val(attr dwarf.Attr) interface{} {
 	return nil
 }
 
-// LoadAbstractOrigin loads the entry corresponding to the
-// DW_AT_abstract_origin of entry and returns a combination of entry and its
-// abstract origin.
-func LoadAbstractOrigin(entry *dwarf.Entry, aordr *dwarf.Reader) (Entry, dwarf.Offset) {
-	ao, ok := entry.Val(dwarf.AttrAbstractOrigin).(dwarf.Offset)
+// LoadAbstractOriginAndSpecification loads the entry corresponding to the
+// DW_AT_abstract_origin and/or DW_AT_specification of entry and returns a
+// combination of entry and its abstract origin.
```

Comment:
Add a note here: "if a DIE has both a specification and an abstract origin the specification will be ignored, the DWARF standard is unclear on how this should be handled".

Reply:
Done!